lib/gotransverse-tract-api/service/service.rb

```diff
@@ -61,6 +61,17 @@ def find_by_billing_account_eid billing_account_eid, options={}
         end
 
         #
+        # @param {Long} original_order_eid
+        # @param {Hash} options
+        #
+        def find_by_original_order_eid original_order_eid, options={}
+          return nil unless original_order_eid.present?
+
+          params ||= GoTransverseTractApi::ApiData.new.get_query_params({original_order_eid: original_order_eid}, options)
+          GoTransverseTractApi.get_response_for(self, params)
+        end
```

Comment:
Never really understood why ||= operator is used in these methods.  Is it even possible that params could already be set?